src/codegen/buildTypeFlag.ts

```diff
@@ -0,0 +1,16 @@
+const buildTypeFlag = process.argv.find(argv => {
```

Comment:
didn't mean to commit this

Reply:
can this be removed so do not cause confusion in the future?